generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -112,11 +112,11 @@ def package_to_apkbuild(ros_distro, package_name, check=True, upstream=False, sr
     ret.append(''.join(['_pkgname=', pkg.name]))
     ret.append(''.join(['pkgver=', pkg.version]))
```

Comment:
add `_gitYYYYMMDD` suffix for upstream build.

> All other suffices are invalid. To package a specific git commit, the date of the commit gets appended to the latest release, e.g. `1.0.0_git20180204`. (https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#pkgver)

Date length looks not limited. `_gitYYYYMMDDHHMMSS` seems allowed.